Startup issue on Wildfly with Java 11 and Spring Boot 2.1.2

The following error occurs in vanilla Wildfly installation with Java 11 and Spring boot `2.1.2`:

```
Caused by: java.lang.IllegalArgumentException: TransformerFactory does not recognise attribute 'http://javax.xml.XMLConstants/property/accessExternalDTD'."}}
```

The issue should be fixed in unreleased version of `spring-ws` project.

See also:
https://stackoverflow.com/questions/54186834/spring-boot-2-1-1-to-2-1-2-error-creating-bean-with-name-payloadrootannotation

We should check this issue after next upgrade of Spring dependencies.

The workaround is to comment out `org.apache.xalan`module in Wildfly configuration.

--- a/pom.xml
+++ b/pom.xml
@@ -32,7 +32,7 @@
     <parent>
         <groupId>org.springframework.boot</groupId>
         <artifactId>spring-boot-starter-parent</artifactId>
-        <version>2.1.2.RELEASE</version>
+        <version>2.1.4.RELEASE</version>
         <relativePath/> <!-- lookup parent from repository -->
     </parent>
 

